packages/suite/package.json

```diff
@@ -21,101 +21,96 @@
         "test-health": "jest -c jest.config.health.js"
     },
     "dependencies": {
-        "@ethereumjs/common": "^2.6.0",
-        "@ethereumjs/tx": "^3.4.0",
+        "@ethereumjs/common": "^2.6.4",
+        "@ethereumjs/tx": "^3.5.2",
         "@fivebinaries/coin-selection": "2.0.0-beta.10",
+        "@hookform/resolvers": "1.3.8",
         "@sentry/integrations": "6.17.2",
         "@sentry/minimal": "6.17.2",
-        "@hookform/resolvers": "1.3.5",
+        "@trezor/components": "*",
+        "@trezor/connect": "9.0.0-beta.1",
```

Comment:
I think we should use "*" for connect in suite packages because that is only way how it should always use latest version from monorepo. In case someone will bump version of connect and forget bump version here, it will use older version from npm and that could lead to confusion. Maybe verify with @szymonlesisz

Reply:
That's correct, I meant only `suite-*` packages that are not published and are builded with webpack.

Reply:
```
error Couldn't find package "@trezor/connect@*" required by "@trezor/suite@1.0.0" on the "npm" registry.
```

any ideas?

Reply:
this is kinda true, afaik @trezor/connect package was not released yet to npm registry :(

Reply:
Unable to do it now. Added note to #5505 

Resolving